lib/widget/bottom_navigation.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class BottomNavigation extends StatelessWidget {
+  const BottomNavigation({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BottomAppBar(
+      shape: CircularNotchedRectangle(),
+      color: Colors.blueGrey,
+      notchMargin: 2.0,
+      clipBehavior: Clip.antiAlias,
+      child: BottomNavigationBar(items: [
+        BottomNavigationBarItem(
+            icon: Icon(Icons.house_rounded), label: "Home"),
+        BottomNavigationBarItem(
+            icon: Icon(Icons.watch_later), label: "Recent"),
+      ],
+          selectedItemColor: Theme.of(context).accentColor
+      ),
+    );;
+  }
```

Comment:
So If I understand, you made the BottomNavigation a widget to reuse it across multiple screens?

Reply:
Yes exactly. I think we remove it and put a centered FAB. As long as we have the back button in the menu we can navigate back to books or home screen